airbyte-integrations/connectors/source-cockroachdb/src/main/java/io/airbyte/integrations/source/cockroachdb/CockroachDbSource.java

```diff
@@ -97,9 +97,4 @@ public static void main(final String[] args) throws Exception {
     LOGGER.info("completed source: {}", CockroachDbSource.class);
   }
 
-  @Override
-  protected JdbcSourceOperations getSourceOperations() {
```

Comment:
how come this was removed? is the class still useful?

Reply:
This method is unnecessary. The `sourceOperations` is an instance variable, and it is assigned in the constructor. The `AbstractJdbcSource` can just reference that variable.